docs/DEPLOYMENT.MD

```diff
@@ -0,0 +1,21 @@
+# DEPLOYMENT
+This document covers steps on setting up this repository on various cloud hosting providers.
+
+# AWS
+
+# Staging
+1. Login to your aws console.
+2. Create a S3 bucket and uncheck `Block all plublic access`.
+3. Enable static website hosting and note down the bucket endpoint. We'll use it to set up a Cloudfront distribution later.
+4. In Cloudfront, create a distribution with the S3 origin, created in the step above.
+5. If you choose to set up a SSL, do so at this step.
+6. Use a DNS provider of your choice. Give an appropriate CNAME and enter the target address. This would be the address of the newly created Cloudfront distribution.
+7. Add the following lines to the `scripts` object in `package.json`.
+    
+    ```"deploy-staging": "vue-cli-service build --mode staging && aws --profile $YOUR_AWS_PROFILE --region $YOUR_AWS_REGION s3 sync ./dist $S3_STAGING_BUCKET_NAME --delete --acl public-read", ```
+    ```"deploy": "vue-cli-service build && aws --profile $YOUR_AWS_PROFILE --region $YOUR_AWS_REGION s3 sync ./dist $S3_PROD_BUCKET_NAME --delete --acl public-read"```
```

Comment:
<img width="1054" alt="Screenshot 2021-08-05 at 4 12 11 PM" src="https://user-images.githubusercontent.com/20169829/128337224-45aa91d7-6639-4a67-9dd9-6a8d9f4a75eb.png">
could be made better
Use this

```json
{
    "deploy-staging": "vue-cli-service build --mode staging && aws --profile $YOUR_AWS_PROFILE --region $YOUR_AWS_REGION s3 sync ./dist s3://$S3_STAGING_BUCKET_NAME --delete --acl public-read",
     "deploy": "vue-cli-service build && aws --profile $YOUR_AWS_PROFILE --region $YOUR_AWS_REGIO s3 sync ./dist s3://$S3_STAGING_BUCKET_NAME --delete --acl public-read"
}
```